x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/io/stream/PlanStreamInput.java

```diff
@@ -180,6 +193,56 @@ public EsqlConfiguration configuration() throws IOException {
         return configuration;
     }
 
+    @Override
+    public int readArraySize() throws IOException {
+        return super.readArraySize();
```

Comment:
leftover?

Reply:
Gotta make it public! I'll leave a comment.

Reply:
I've reworked it so I don't need to make this public.